plugin/adsk/scripts/mayaUsdMenu.mel

```diff
@@ -109,6 +109,24 @@ proc string mayaUSD_getSelectionKind() {
     return $selectionKind;
 }
 
+///////////////////////////////////////////////////////////////////////////////
+// mayaUSD_setPointInstancesPickMode
+// Helper function to set the point instances pick mode.
+global proc mayaUSD_setPointInstancesPickMode(string $mode) {
+    optionVar -stringValue "mayaUsd_PointInstancesPickMode" $mode;
+}
+
+///////////////////////////////////////////////////////////////////////////////
+// mayaUSD_getPointInstancesPickMode
+// Helper function to get the point instances pick mode.
+proc string mayaUSD_getPointInstancesPickMode() {
+    string $mode = "";
+    if (`optionVar -exists "mayaUsd_PointInstancesPickMode"`) {
+        $mode = `optionVar -q "mayaUsd_PointInstancesPickMode"`;
+    }
+    return $mode;
+}
```

Comment:
New helper functions to deal with set/get of the point instances pick mode. Very similar to the `SelectionKind` ones just above.